### PR TITLE
Make the Readme file more valuable

### LINKED ISSUE #7: Our Other issue

### LINKED ISSUE #8: Our Re-Other Issue

### PR DESCRIPTION
**Pull request by ghost**
Create a much better Readme layout and we better inform people about what our repository is all about.

Fixes #7

### CODE REVIEW

File: README.md
Comment (Tux73) [inline README.md:1]: Looks good to me

File: ReadmeStructure.md
Comment (Tux73) [inline ReadmeStructure.md:1]: I dont't like the color grren.

**Review by Tux73 — CHANGES_REQUESTED** [review]
Address what I said.

File: ReadmeStructure.md
Comment (kappesLab) [reply ReadmeStructure.md:1]: Too bad. That is how new lines are indicated

**Review by ghost — COMMENTED** [review]
I think i am done.

**Review by Tux73 — APPROVED** [review]
Ok, fine. It looks good now.

**Comment by kappesLab** [comment]
Fixes #8

**Pull request merged**